app/controllers/foo_controller.rb

```diff
@@ -0,0 +1,13 @@
+class FooController < ActionController::Base  
+  def new_user
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
```

Comment:
__[rails_best_practices]__
* move finder to named_scope [:ref](http://rails-bestpractices.com/posts/2010/07/14/move-finder-to-named_scope/)